tests/unit/xoopsLib/Xoops/LocaleTest.php

```diff
@@ -104,4 +104,30 @@ public function test_getUserLocales()
         $this->assertTrue(is_array($locales));
         $this->assertTrue(in_array('en_US', $locales));
     }
+
+    public function test_normalizeLocale()
+    {
+        $class = $this->myClass;
+
+        $locale = 'en-latn-us';
+        $actual = $class::normalizeLocale($locale, '_', false);
+        $this->assertEquals('en_US', $actual);
+        $actual = $class::normalizeLocale($locale, '-', false);
+        $this->assertEquals('en-US', $actual);
+        $actual = \Xoops\Locale::normalizeLocale($locale, '_');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Xoops\Locale' in method 'test_normalizeLocale'.](https://www.codacy.com/app/richard_3/XoopsCore/file/2433843220/issues/source?bid=3194562&fileBranchId=3242327#l117)